improver_tests/set_up_test_cubes.py

```diff
@@ -76,16 +76,14 @@ def construct_yx_coords(
         raise ValueError("Grid type {} not recognised".format(spatial_grid))
 
     if grid_spacing is None:
-        if domain_corner is None:
-            y_array, x_array = _default_grid(ypoints, xpoints, spatial_grid)
-        else:
-            raise ValueError("Grid spacing required to setup grid from domain corner.")
-    else:
-        if domain_corner is None:
-            domain_corner = _set_domain_corner(ypoints, xpoints, grid_spacing)
-        y_array, x_array = _create_yx_arrays(
-            ypoints, xpoints, domain_corner, grid_spacing
-        )
+        if spatial_grid == "latlon":
+            grid_spacing = 10
+        elif spatial_grid == "equalarea":
+            grid_spacing = 2000
```

Comment:
You could add these to the centralised dictionary.

Reply:
Good point!

Update in commit 63973dfd71cda592537f7e66c667547619ae8271.